keystone_api/apps/users/views.py

```diff
@@ -0,0 +1,49 @@
+"""Application logic for rendering HTML templates and handling HTTP requests.
+
+View objects handle the processing of incoming HTTP requests and return the
+appropriately rendered HTML template or other HTTP response.
+"""
+
+from rest_framework import permissions, status, viewsets
+from rest_framework.response import Response
+
+from .models import *
+from .serializers import *
+
+__all__ = [
+    'ResearchGroupViewSet',
+    'UserViewSet',
+]
+
+
+class ResearchGroupViewSet(viewsets.ModelViewSet):
+    """View or create ResearchGroups."""
+
+    permission_classes = [permissions.IsAdminUser]
+    serializer_class = ResearchGroupSerializer
+    filterset_fields = '__all__'
+
+    def create(self, request, *args, **kwargs) -> Response:
```

Comment:
The `ModelViewSet` base class already provides create functionality ([docs](https://www.django-rest-framework.org/api-guide/viewsets/#modelviewset)). Is there a reason it is overwritten here?

Reply:
Honestly I was just trying not to stray from how I saw it being set up for the `AllocationRequestReviewViewSet`. I can see why there was an override there now.